### PR TITLE
LPD 28817 addig new blacklisted tests

### PR DESCRIPTION
**Pull request by ZakiMiklos**
https://liferay.atlassian.net/browse/LPD-28817

### CODE REVIEW

File: modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java [hunk 1]
```diff
@@ -656,6 +663,7 @@ else if (Objects.equals(
 		"com.liferay.object.admin.rest.dto.v1_0.ObjectRelationship",
 		"com.liferay.object.admin.rest.dto.v1_0.ObjectValidationRule",
 		"com.liferay.object.admin.rest.dto.v1_0.ObjectView",
+		"com.liferay.portal.language.rest.dto.v1_0.Message",
```
Comment (ZakiMiklos) [inline modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java:666]: I am not 100% sure that it should be added. It gets an error but no error message

**Comment by 4lejandrito** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/152958. I removed some unrelated commits on top of yours that I think you added by accident.

File: modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java (hunk 1 above)
Comment (4lejandrito) [reply modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java:666]: Let's disable it for now. We will be soon working on making this test better.

**Comment by 4lejandrito** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/152959. I had to change the LPS of the commit to point to the subtask. Always use the subtask when developing a task.

**Pull request closed without merging**